docs/docs/quickstart.mdx

```diff
@@ -108,7 +107,7 @@ const pathPrefix = '/.ory/kratos/public';
 app.use(pathPrefix + '/', (req, res) => {
   const url = 'https://public.ory-kratos' + req.url.replace(pathPrefix, '');
 
-  // Uses the request library to forward the request to ORY Kratos
+  // Uses the `request` library to forward the request to Kratos
```

Comment:
We use `ORY` as a prefix consistently across all docs, product pages, manuals - this shouldn't be the odd one out so please revert it :)

The prefix `ORY` is important because there are other projects called e.g. Hydra and other things (e.g. Game of Thrones) for Oathkeeper.

Reply:
Okay, sounds good. I'll revert it.

However, just as a suggestion I would consider thinking about this in the future since it sounds like it's just SEO related and I'm guessing searching for "ORY Kratos" or "Kratos identity management" would probably bring up ORY Kratos and not God of War, even without using "ORY" everywhere "Kratos" is used.